docs/content/docs/recipes/teams.md

```diff
@@ -0,0 +1,347 @@
+
+```ts {linenos=inline,hl_lines=[134,136,137]}
+const onSaveHandler = async (saveEvent: microsoftTeams.settings.SaveEvent) => {
+    const host = "https://" + window.location.host;
+    const containerId = await createContainer();
+    microsoftTeams.settings.setSettings({
+        contentUrl: host + "/<your tab name>/?" + containerIdQueryParamKey + "=" + containerId + "&name={loginHint}&tenant={tid}&group={groupId}&theme={theme}",
+        websiteUrl: host + "/<your tab name>/?" + containerIdQueryParamKey + "=" + containerId + "&name={loginHint}&tenant={tid}&group={groupId}&theme={theme}",
+        suggestedDisplayName: "<your tab name>",
+        removeUrl: host + "/<your tab name>/remove.html?theme={theme}",
+        entityId: entityId.current
+    });
+    saveEvent.notifySuccess();
+};
+```
+
+Please make sure to replace `<your tab name>` with the actual tab name from your project.
+
+{{< callout warning >}}
+
+Since the content page URL is used to store the container ID, this record will be removed if the Teams tab is deleted.
+Additionally, every content page can only support one container ID.
+
+{{< /callout >}}
+
+### Refactor content page to reflect Fluid application
+
+Open the file `src/client/<your tab name>/<your tab name>.tsx` in your code editor. A typical Fluid-powered application consists of a view and a Fluid data structure. Let's just focus on getting/loading the Fluid container and leave all the Fluid related interactions in a React component.
+
+1. Add the following import statements in the content page.
+
+```ts
+import { IFluidContainer } from "fluid-framework";
+import { getContainer, containerIdQueryParamKey } from "./Util";
+```
+
+1. Now remove all the code below the import statements inside the content page and replace it with the following. Make sure to replace <your tab name> with the tab name you defined for your project.
+
+```ts
+export const <your tab name> = () => {
+  // TODO 1: Initialize Microsoft Teams.
+  // TODO 2: Initialize inTeams boolean.
+  // TODO 3: Define container as a React state.
+  // TODO 4: Define a method that gets the Fluid container
+  // TODO 5: Get Fluid container on content page startup.
+  // TODO 6: Pass the container to the React component as argument.
+}
+```
+
+1. Replace `TODO 1` with the following code. For the content page to display in Teams, you must include the [Microsoft Teams JavaScript client SDK](https://docs.microsoft.com/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true) and include a call to initialize it after your page loads.
+
+```ts
+microsoftTeams.initialize();
+```
+
+1. Replace `TODO 2` with the following code. Because Teams application is just an IFrame injection of a webpage, you need to initialize the `inTeams` boolean constant in order to know if the application is inside Microsoft Teams or not, and if the Teams resources, such as the `contentUrl`, are available.
```

Comment:
```suggestion
1. Replace `TODO 2` with the following code. Because the Teams application is just an IFrame injection of a webpage, you need to initialize the `inTeams` boolean constant in order to know if the application is inside Microsoft Teams or not, and if the Teams resources, such as the `contentUrl`, are available.
```